boards/nordic/nrf54h20dk/nrf54h20dk_bicr.dtsi

```diff
@@ -24,7 +24,7 @@
 		lfosc-mode = "crystal";
 		lfosc-loadcap = <15>;
 
-		lfrc-autocalibration = <20 40 3>;
+		lfrc-autocalibration = <16 0.5 2>;
```

Comment:
in my opinion it should be: `<16 2 2>` based on:
```
  lfrc-autocalibration:
    type: array
    description: |
      A list of values pertaining to LFRC autocalibration settings. The prop
      is encoded as <temp-interval temp-delta interval-max-count>, where:
      - temp-interval is the temperature measurement interval in 0.25s steps
      - temp-delta is the temperature delta that should trigger a calibration
        in 0.25 degree steps
      - interval-max-count are the max number of temp-interval periods in
        between calibrations, independent of temperature changes.
```
and based on this PR description